service/parserprovider/default.go

```diff
@@ -18,5 +18,5 @@ package parserprovider
 // defined by the --config command line flag and overwrites properties from --set
 // command line flag (if the flag is present).
 func Default() ParserProvider {
-	return NewSetFlag(NewFile())
+	return NewMergeProvider(NewFile(), NewSetFlag())
```

Comment:
Is the merge bahavior already covered by unit tests, or should we add new ones for `NewMergeProvider` explicitly?

Reply:
Done. Added tests that were not covered in the Default tests.